fast_test.py

```diff
@@ -1,12 +1,14 @@
 # THIS IS A DEVELOPMENT PERFORMANCE TESTING CLASS, REMOVE WHEN NO LONGER NEEDED
```

Comment:
This file should not be included in the commit, it isn't an official part of the repo, I don't know how it got into main.

Reply:
Removed.